File: Cap 10/Topico 10.7/verdados/verifdados.pl
=pod

 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 10 - Topico 10.7
 *
 * Programa ..: OPERACAO COM SUBROTINA DE FUNCAO

=cut

use strict;
use warnings;

sub IGUAL {
  my ($A, $B) = @_;
  return $A == $B;
}

my $X;
my $Y;

print "VERIFICACAO DE DADOS\n";
print "FUNCAO SIMPLES COM PASSAGEM DE PARAMETRO\n\n";
print "Informe o 1o. valor: ";
$X = <STDIN>;
chomp($X);
print "Informe o 2o. valor: ";
$Y = <STDIN>;
chomp($Y);
print "\n";
if (IGUAL($X, $Y)) {
  print "Valores sao iguais\n";
} else {
  print "Valores sao diferentes\n";
}

File: Cap 10/Topico 10.7/verdados/verifdados.go
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 10 - Topico 10.7
 * 
 * Programa ..: OPERACAO COM SUBROTINA DE FUNCAO
 */

package main

import (
  "fmt"
)

func IGUAL(A, B int) bool {
  return A == B
}

var (
  LIMITE int
  RESP   int
)

func main() {
  fmt.Println("VERIFICACAO DE DADOS")
  fmt.Println("FUNCAO SIMPLES COM PASSAGEM DE PARAMETRO")
  fmt.Println()

  fmt.Print("Informe o 1o. valor: ")
  fmt.Scanln(&LIMITE)
  fmt.Print("Informe o 2o. valor: ")
  fmt.Scanln(&RESP)
  fmt.Println()

  if IGUAL(LIMITE, RESP) {
    fmt.Println("Valores sao iguais")
  } else {
    fmt.Println("Valores sao diferentes")
  }
}
